src/mtype.d

```diff
@@ -4747,7 +4747,7 @@ extern (C++) final class TypeSArray : TypeArray
             if (d1 != d2)
             {
             Loverflow:
-                error(loc, "%s size %llu * %llu exceeds 16MiB size limit for static array", toChars(), cast(ulong)tbn.size(loc), cast(ulong)d1);
+                error(loc, "Static array size too big", toChars(), cast(ulong)tbn.size(loc), cast(ulong)d1);
```

Comment:
@JohanEngelen Sorry, I shouldn't made the PR late midnight...
I can replace that part with something like this, but it might be unnecessary code duplication:
```D
bool overflow = false;
auto static_arr_size = mulu(tbn.size(loc), d2, overflow);
error(loc, "%s static array size %llu * %llu overflowed to %llu", toChars(), cast(ulong)tbn.size(loc), cast(ulong)d1, cast(ulong)static_arr_size);
goto Lerror;
```
What do you suggest?